### PR TITLE
Update rulesets for v2.0

### PR DESCRIPTION
**Pull request by xyhuang**
- Added v2.0 ruleset
- Moved verification and packaging scripts to /scripts

### CODE REVIEW

**Comment by xyhuang** [comment]
Addresses #207

**Review by emizan76 — COMMENTED** [review]
Can you add verify_for_v2.0_training.sh and change README.md?

File: mlperf_logging/compliance_checker/training_2.0.0/common.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+- KEY:
+    NAME:  submission_platform
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] != '' "
+
+- KEY:
+    NAME:  submission_division
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] in ['closed', 'open'] "
+    POST:  " enqueue_config('training_1.1.0/{}_common.yaml'.format(v['value'])) "
```
Comment (shangw-nvidia) [inline mlperf_logging/compliance_checker/training_2.0.0/common.yaml:44]: `raining_2.0.0` ?

**Comment by xyhuang** [comment]
> Can you add verify_for_v2.0_training.sh and change README.md?

added, btw i also moved all the verify scripts to the /scripts sub dir, pls let me know if there's any concern.

**Comment by xyhuang** [comment]
one another note: I did not change the following 2 files for ssd as they were introduced in a previous commit, pls let me know if any change is needed in those files:
- `mlperf_logging/compliance_checker/training_2.0.0/closed_ssd.yaml`
- `mlperf_logging/rcp_checker/training_2.0.0/rcps_ssd.json`
cc: @emizan76 @shangw-nvidia

**Comment by shangw-nvidia** [comment]
> on another note: I did not change the following 2 files for ssd as they were introduced in a previous commit (#209, #211), pls let me know if any change is needed in those files:
> 
> * `mlperf_logging/compliance_checker/training_2.0.0/closed_ssd.yaml`
> * `mlperf_logging/rcp_checker/training_2.0.0/rcps_ssd.json`
> 
> cc: @emizan76 @shangw-nvidia

Those two files look fine to me; I don't see any (obvious) issues. Once this PR (and a few others that @emizan76 submitted) gets merged in, we can test it on the v1.1 submission to make sure.

**Review by shangw-nvidia — APPROVED** [review]

**Review by emizan76 — APPROVED** [review]

**Pull request merged**